site/docs/how-to-vote.md

```diff
@@ -0,0 +1,75 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Voting on an Iceberg Release
+
+Before a new version of Apache Iceberg is released, a release candidate will be prepared and a vote
+will occur. Announcements will be made on the Iceberg [developer's mailing list](https://iceberg.apache.org/#community/#mailing-lists)
```

Comment:
I think it's good to avoid passive voice in documentation because it is more direct. I like the background here, but I think we can have both. How about something like this?

> Each Apache Iceberg release is validated by the community by holding a vote. A community release manager will prepare a release candidate and call a vote on the Iceberg dev list. To validate the release candidate, community members will test it out in their downstream projects and environments.

> In addition to testing in downstream projects, community members also check the release's signatures, checksums, and license documentation.

Reply:
Thanks @rdblue ! Updated the PR to use your suggestion for the first paragraph.